.circleci/config.yml

```diff
@@ -208,11 +208,11 @@ jobs:
           command: |
             # --no-sandbox becasue we are running as root and chrome requires
             # this flag for now: https://crbug.com/638180
-            CHROME_FLAGS_BASE="--no-first-run -start-maximized --no-sandbox"
+            CHROME_FLAGS_BASE="--no-first-run -start-maximized --no-sandbox --use-gl=swiftshader"
```

Comment:
With firefox we use gallium softpipe to render, and it seems to do all the GL tests properly. Could the same work here, and allow fewer tests to be disabled? Or is swiftshader necessary for some reason?

Reply:
Yes, we should probably switch using the gallium softpipe.   The doesn't work with xvfb though sadly, as I discovered yesterday.   So we can switch away from xvfb as a seperate change lager perhaps?   I the mean time there is no harm in tagging the tests and don't work with switftshader.  Its probably worth opening bug reports for them too.